packages/dcos-integration-test/extra/test_service_discovery.py

```diff
@@ -7,7 +7,7 @@
 MESOS_DNS_ENTRY_UPDATE_TIMEOUT = 60  # in seconds
 
 
-def _service_discovery_test(cluster, docker_network_bridge=True):
+def _service_discovery_test(rcluster, docker_network_bridge=True):
```

Comment:
We already have a pytest fixture with the same name and currently it works because we supply this argument when this method. Keeping it this way is error-prone and risky.